images/dex/tests/runs.sh

```diff
@@ -5,13 +5,10 @@ set -o errexit -o nounset -o errtrace -o pipefail -x
 PORT=5556
 
 URL="http://127.0.0.1:${PORT}"
```

Comment:
It looks like this runs on the host below
```suggestion
URL="http://127.0.0.1:${FREE_PORT}"
```